worldedit-core/src/main/java/com/fastasyncworldedit/core/Fawe.java

```diff
@@ -346,4 +347,46 @@ public Thread setMainThread() {
         return this.thread = Thread.currentThread();
     }
 
+    /**
+     * Non-api. Handles an input FAWE exception if not already handled, given the input boolean array.
+     * Looks at the {@link FaweException.Type} and decides what to do (rethrows if we want to attempt to show the error to the
+     * player, outputs to console where necessary).
+     *
+     * @param faweExceptionReasonsUsed boolean array that should be cached where this method is called from of length {@code
+     *                                 FaweException.Type.values().length}
+     * @param e                        {@link FaweException} to handle
+     */
+    public static void handleFaweException(boolean[] faweExceptionReasonsUsed, FaweException e) {
+        FaweException.Type type = e.getType();
+        switch (type) {
+            case OTHER:
+                LOGGER.warn("FaweException: " + e.getMessage());
```

Comment:
Why do we only log the message instead of the actual exception?

Reply:
This probably ought to be a stacktrace yeah